libsession/src/main/java/org/session/libsession/messaging/utilities/UpdateMessageBuilder.kt

```diff
@@ -0,0 +1,99 @@
+package org.session.libsession.messaging.utilities
+
+import android.content.Context
+import org.session.libsession.R
+import org.session.libsession.messaging.MessagingConfiguration
+import org.session.libsession.utilities.ExpirationUtil
+import org.session.libsignal.service.api.messages.SignalServiceGroup
+
+object UpdateMessageBuilder {
+
+    fun buildGroupUpdateMessage(context: Context, updateData: UpdateMessageData, sender: String? = null, isOutgoing: Boolean = false): String {
+        val updateType = updateData.type
+        var message: String = ""
+        if (!isOutgoing && sender == null) return message
+        val senderName: String? = if (!isOutgoing) {
+            MessagingConfiguration.shared.storage.getDisplayNameForRecipient(sender!!) ?: sender
+        } else { sender }
```

Comment:
Should we maybe make this say "You" if the message is outgoing?

Reply:
Yeah, makes sense, I'll correct that